lib/src/http/vm/request_mixin.dart

```diff
@@ -42,75 +42,76 @@ abstract class VMRequestMixin implements BaseRequest, CommonRequest {
   @override
   void cleanUp() {
     if (_isSingle) {
-      _client?.close();
+      _client.close();
     }
   }
 
   @override
   Future<Null> openRequest([Object? client]) async {
```

Comment:
can this be HttpClient? and still override the super class?

Reply:
Doesn't look like it, unfortunately